seed/views/organizations.py

```diff
@@ -195,6 +195,79 @@ def _save_fields(org, new_fields, old_fields, is_public=False):
 _log = logging.getLogger(__name__)
 
 
+class RulesSubSerializer(serializers.Serializer):
```

Comment:
FYI, the existing serializers all live under seed/serializers, rather than views. (I've been using them for all the API endpoint I've been writing)